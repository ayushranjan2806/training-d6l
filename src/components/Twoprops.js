import React from 'react'

export const Twoprops = ({message1 ,message2}) => {
  return (
    <div>

      <div>{message1}</div>

    <div>{message2}</div>
    </div>
  )
}
