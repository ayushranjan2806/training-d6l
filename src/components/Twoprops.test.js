import React from 'react';
import renderer from 'react-test-renderer'



import { Twoprops } from './Twoprops';

test('Message snapshot with differet',()=>{
    const component = renderer.create(<Twoprops message1="click " message2="ayushddf"/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot(); 
})