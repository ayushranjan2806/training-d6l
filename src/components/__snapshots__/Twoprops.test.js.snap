// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message snapshot with differet 1`] = `
<div>
  <div>
    click 
  </div>
  <div>
    ayushddf
  </div>
</div>
`;
